src/config.h

```diff
@@ -32,9 +32,14 @@ struct GatewayConfig {
 };
 
 struct NetworkConfig {
-  std::string socket_commands_path{"/tmp/sota-commands.socket"};
-  std::string socket_events_path{"/tmp/sota-events.socket"};
-  std::vector<std::string> socket_events{"DownloadComplete", "DownloadFailed"};
+  NetworkConfig() : socket_commands_path("/tmp/sota-commands.socket"), socket_events_path("/tmp/sota-events.socket") {}
+  std::string socket_commands_path;
+  std::string socket_events_path;
+  std::vector<std::string> socket_events = {"DownloadComplete", "DownloadFailed"};
```

Comment:
Is there a reason to revert these changes, or is this a rebase error?

Reply:
No sorry, I did not noticed that when resolving conflicts for rebase

Reply:
Heh, no worries

Reply:
Can you un-revert these back to the c++11 style, please?

Reply:
Sorry, I marked this change in my mind as resolved. 10 minutes and will update PR